crates/forge-runner/src/gas.rs

```diff
@@ -88,3 +88,21 @@ fn get_l1_gas_usage(
     message_segment_length * eth_gas_constants::SHARP_GAS_PER_MEMORY_WORD
         + onchain_data_segment_length * eth_gas_constants::SHARP_GAS_PER_MEMORY_WORD
 }
+
+// TODO::copied from blockifier, because it became private
```

Comment:
Let's create an issue for making it public in blockifier and link its number in the comment.

Reply:
cc @tkumor3